bucket/monokle.json

```diff
@@ -0,0 +1,42 @@
+{
+    "version": "1.4.0",
+    "description": "K8s tool for creating, validating, debugging and managing manifests.",
+    "homepage": "https://kubeshop.github.io/monokle",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/kubeshop/monokle/releases/download/v1.4.0/Monokle-win-1.4.0-x64.exe#/dl.7z",
+            "hash": "sha512:1c5537e5dbb1ea688bf6cd7544e53259a97825a12a27029befabc1fedd908b8439cf80017dd72c5f5ce5181694ba1915520d0930616898b24f2671e8f07b0303",
+            "pre_install": [
+                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstall*\" -Force -Recurse"
+            ]
+        }
+    },
+    "bin": "Monokle.exe",
+    "shortcuts": [
+        [
+            "Monokle.exe",
+            "Monokle"
+        ]
+    ],
+    "depends": [
+        "kubectl",
+        "helm",
+        "kustomize"
+    ],
```

Comment:
Yes, during startup these dependencies checks during monokle startup and warning show in logs and UI
If this utils missed, part of function is unavailable

I can move helm and kustomize into "suggest", but not sure it will right choice

Reply:
Only put apps that are absolutely required in `depends`, i.e. without which Monokle will not function at all. Put the rest in `suggest`.

Reply:
kubectl is necessary, `helm` and `kustomize` moved to `suggest`